components/breadcrumbs/default.htm

```diff
@@ -1,11 +1,11 @@
 {% set crumbOptions = __SELF__.options %}
 {% if currentCrumb.show_crumb %}
-<ol class="{{crumbOptions['main-ol-class']}}">
+<ol itemscope itemtype="https://schema.org/BreadcrumbList" class="{{crumbOptions['main-ol-class']}}">
 {% set activeCrumb = '' %}
 {% for crumb in breadcrumbs %}
     {% if crumb.in_crumb_trail %}
-    <li class="{{crumbOptions['main-li-class']}} {% if crumb.crumb_disabled %} {{crumbOptions['disabled-class']}} {% endif %} {% if loop.last %} {{crumbOptions['active-class']}} {% endif %}" {% if loop.last %} id="bcLast" {% endif %}>
-        {% if not loop.last and not crumb.crumb_disabled %} <a href="{{crumb.baseFileName | page}}"> {% endif %} {{crumb.title}} {% if not loop.last and not crumb.crumb_disabled %}</a>{% endif %}
+    <li {% if not loop.last and not crumb.crumb_disabled %} itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement" {% endif %} class="{{crumbOptions['main-li-class']}} {% if crumb.crumb_disabled %} {{crumbOptions['disabled-class']}} {% endif %} {% if loop.last %} {{crumbOptions['active-class']}} {% endif %}" {% if loop.last %} id="bcLast" {% endif %}>
```

Comment:
@SergeyOsipoff is there any reason to do not add microdata for last element? 
I don`t  know, what do You think?
There are some questions like [this](https://stackoverflow.com/questions/33688608/how-to-markup-the-last-non-linking-item-in-breadcrumbs-list-using-json-ld). May be we can add microdata with current page url, or ommit last item, because 

> The breadcrumb trail may include or omit a breadcrumb for the page on which it appears.

Reply:
As it turned out, this is a controversial issue. However, there is one point that says that we should not mark the last item - the text of the last paragraph can be changed by JS. In this case, the robot that will read the microcode will not see this change and the search results will not show what we would like to see there.

Very curious what exactly there will be displayed. I'm now waiting for the results of indexing my site with a version of crumbs with the last item withoud microcode.

Sorry for my English. This is Google Translate :) 

And, Nikolay, apparently we could also speak in Russian :)

By the way, from the point of view of JS's influence, one more minor revision is needed. I'll send it a little later if we decide not to mark the last item.

Reply:
> the text of the last paragraph can be changed by JS

I think, You are right, it will be be better to leave last item without microdata.